Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -238,6 +238,20 @@ static bool exceedsRenderTreeSizeSizeThreshold(uint64_t thresholdSize, uint64_t
     return WebPasteboardProxy::singleton().grantAccessToCurrentData(m_legacyMainFrameProcess, pasteboardName, WTFMove(completionHandler));
 }
 
+void WebPageProxy::grantAccessToCurrentPasteboardData(const String& pasteboardName, std::optional<FrameIdentifier> frameID)
+{
+    if (!hasRunningProcess())
+        return;
+    if (frameID) {
```

Comment:
This check isn't needed.  Just pass the optional to WebFrameProxy::webFrame without dereferencing.  It'll make it a little more concise.